qc.ipynb

```diff
@@ -0,0 +1,619 @@
EYgNIBZuo/ZgMAkpYJS3r/YFm2+kM2ZRn6e3ffu1+Z2+5xKy55DRsfORYAcnfGdePTQLfbYHdcl3/eZ9b5oqY6EwAdYAGqrFU6P7VuUDp03krdGfajGyozusSma2+5E39XXH9H3PbKb9O5y3OYrUDE+e1vWICV78zlK5L6j4/Nt0/ByB85QeMNs/Qx0gNP1xnGNwAAQ3Oce+Pdk50TSq8WOCpR9dgEAEAkasFwXgPTsAGnmnXr5ddj+C/uDByj4trViO7xikXcJWzqLpqsu742DzBQsEs9mIZhUpeA9tts6UAsUeJ0hX0tgbqRan6mkVQBFQDYEbUkDAAUV7XCTFjO8xTVc0NbDlGlsfkf7dVdbeXuT/U8Uxk1sXe4GlPy4m7IfHWe5i8ejfzn3gAAfHKBeh8XfpwCpApGpSFgR9VJ6ssMFO5Qb8qdJwkMeU5dYL+NNnZNcF67Pur/hTts1I1Sz03/9TbMFnWdeXskEq3ucjYxDLpXvXa1t01B4UdOAHqYenz93ox6y/hEgNa+aju2x0DiCPVHUPz9fDQPdf5wSgkvGE1FYRU5z1OteqyxAS2I5quxyb0xiJbcS9EQERFR92MJLhEREREREXULZkDpgBQ1LcBysiSy7dKdI56ch/fP/SkAYNP5P8bIx3/u3ejcddOjx2Qvj/B3xnXLcQ3ocly3FBcINszxlx3ptT/TDi1zbLmlucPmOt1ff+xlSN11M9vSkcZE/lLb3ZdP0Rc54NfBbGj1THUNbtmyNL3HaMWhS2OtPENnuGTEywq72U9AldUCThmy89y0tQ5q1W1TnPNI2M6akDCdk0ekvua8ogSGfuOtjPtv+/04DD//TfXNN4AtV80O3D4qsShwLrdZUvHvVuty4f7vAXtGRpzHaOvHs/EXkxDdo16/lNNwadu0GAwnmRdpEDCcSzUSQHyPuv7tp8Uw/I8qbRzdm0RjucrutRY5nXaPKEHB2H4AnKZOzvNUWG3BbFXnqTl9CPI/VQdP9DXR+C2V+Ww81MleVkt8eoQqXy1d04Q9o1VqPFkINB6qjnfYP2xEmp1j9DeRLFbbTU6HrL4f2ojVq5PvPcxAw5EqhTvm8jVodcq16z5nYZDz/PUduxutnw4EAOR/pJ4vOwq0OI2JpACSZerJieSlEH9PddhtHpqEaHYaGZU3Itmo0usyKmEUqvHSrW5ImTAM9TyafZK6RHfY0tux9VKW4RIREfW0DmdAX3jhBZx99tkoLy+HEAJPP/104HYpJebOnYvy8nLk5+fjlFNOwTvvvBMYk0gkcM0112DgwIEoLCzEOeecg+3bt3+mB0JERERERES9W4czoI2NjZgwYQIuu+wyfO1rX8u4feHChVi0aBGWLl2KMWPG4NZbb8UZZ5yB9evXo6hITdqdMWMG/vjHP+Lxxx/HgAEDcN111+Gss87C2rVrYZqcr0NdJ2JmTWMGvtt8wU0AgJGP/xzCyXBKXyZTmOGWZgEyl2fx9ustndH0J2r9GUE3ywaorKB/zczgsVQ2NEwWdF+kNx5y1VxZqTN6ZYu9JWDcqZc2oLOQqXwv6ylSatkTl/s4TW91kTYzny7baWBjx23AyYCKiPp/JGbBdp7YWDSF2j9XqG3T8jLjjdBzQ/1G3akyn8KGfp1H3LgKW36mrlOa0G+dPYcDySJnkCGx8d5JAICKq17OOO7mhVN0RjhRInWTJLPBQHOZuqYRc7y1WFsGCPRfr5o87KlQtzcNTaFs8SsAgE/unKyP3XBY8L3cXKtOVLhTYs+paj5lZKPKKtaNiKJxqDp31eB8FG9yljlJAa3F6jwfniMx/En1BovV2xjxtBq/dbo6z8fHm+izTd2vfhRQ/EZMnzvlNKHqc2g9Pv2TWtO0vr4AcLKdxRvUuKbBAvHdaqwdA5L9nEZGBTaaD3PeWJaALHDmen5YqJ8zGbUh6pxlYkx13KL+jUgkneyqLRAb1KS3iYiIqOd1OACdPn06pk+fnvU2KSWWLFmCm266Ceeeey4A4JFHHkFpaSmWLVuGK664AnV1dXjooYfw6KOP4vTTTwcA/Pa3v8WQIUPw97//HWeeeeZneDh0sFtxmmogU/lcsNTOdEry3MZDYWy+4KZAOa4ORrP9IZteihuyIZG/2ZCwhe4Gu3mht/7miBtWoWpBMAgFvOZEwkbauqP7pnqGOl7ZEi/oq7myEoPuyx4E7v7uFOf8Eofcp67742vUMYyk1+zItLzg2op79/eXIVfNn6K/X/9T1fn28HmLA42Wchk1ezUAYMN9J+jA03TWco3FU4hFVPDWJ96K/DOr9P0+eeZwAMCQr7+tA9Bh/3O77pbqNooaef1qfZ9tN1fq12X4zSuxeaFT/lsQ/FDASKgH7G+utOFBp6Oh5XvgQgLOBwojr34ZG/5nohr7PxMx5ttOU6Ylk1H0oTpe0VZ18mSRqcrCAditKcD5gES0GjrYlxEb9qHqsbcebcOqcxofxdT5DvnlKrQ45dvJvhINTkcnOwZYfZ0uvfkpNJapJkSN5QLNR6guQ8Wr1RvOTEjkf6quKdHPROl/e6+Tu71x1GQMnbAVgGoCNmzSTgDA22Kk8xzYiDSp40XrgegeFfCKAYBw1m2VTRGYTpfHIyq2Y0e9quPe+14J7HKn85FTgluc34KdLep207SRSqrjRWMpHPHkPADQJfdERETU/Tq1CVFVVRWqq6sxbdo0vS8ej2Pq1KlYuVL9MbJ27Vokk8nAmPLycowbN06PISIiIiIiogNPpzYhqq5W69KVlpYG9peWlmLr1q16TCwWQ//+/TPGuPdPl0gkkEh4dXn19eHX/qGD08ppt2dkQT8LYcpAGa6WI9MZLLfN3ZAomL2U+MAppxQ2dLMbfzbUX27rHtdIBY/tb0qkr993ff6y1h2zK72mRL6HV3O1Ooa7fIbLXQNz4K9WBZZrcZXeo8Z/fI1XlmvHoPsoCQkIdxkyKXSG00z4spw3OfeLeOXJALJmQ6vmT9GZViFtyKR6Mk
```

Comment:
overal = overall

(I tried making this a proper suggestion through GitHub and madness ensued.)